src/Components/QuestionList/QuestionViewTypes/SmallCardList/SmallCardList.tsx

```diff
@@ -0,0 +1,28 @@
+import { ListGroup } from "react-bootstrap";
+import { Question } from "../../../../Apis/types";
+import { cardOrder } from "../../../../Apis/types";
+import { SmallCard } from "./SmallCard/SmallCard";
+import "./SmallCardListS.scss";
+
+interface Props {
+    currentQuestions: Question[];
+    showDelete: boolean;
+}
+
+export const SmallCardList = ({ currentQuestions, showDelete }: Props) => {
+    return (
+        <ListGroup className="SmallCardList">
+            {currentQuestions
+                .sort(
+                    (a, b) => cardOrder(a.difficulty) - cardOrder(b.difficulty)
+                )
+                .map((question) => {
```

Comment:
we should make the sort function, a utils as we have used it now in 3 different places